modules/actor/mixin.js

```diff
@@ -195,4 +201,6 @@ export default WFRP4eDocumentMixin = (cls) => class extends cls {
         }
     }
 
-}
\ No newline at end of file
+}
+
+export default WFRP4eDocumentMixin;
```

Comment:
Does separating this help JSDoc in any way?

Reply:
No, but it removes the IDE's `Unresolved variable` error. I can revert it as it doesn't stop the code from executing. But then again, this header would work just as fine:
```js
export default (cls) => class extends cls {
```
I don't really have a preference on this, I'm fine either way

![image](https://github.com/moo-man/WFRP4e-FoundryVTT/assets/11304207/2c796d28-4e24-4215-ae5f-92275a63b644)